### ISSUE TITLE
Whats gooing on with the Train Schedule? 

### ISSUE THREAD

**Issue by olathordolsson** [HannahMitt/HomeMirror#48]
I find the moduel "trainScheduelModul.java" but it seems its not initiated anywhere?

Some thing your working on and haven't finished or is it my skills thats lacking =)

/ Ola

**Comment by HannahMitt** [comment]
Right, thats actually dead code. Should probably just delete and leave on a branch :)

That code is actually running on a Raspberry Pi in my home, and controlling a Lifx bulb. It changes color based on whether its a good time to catch the train. Shoved it in the project but never actually used it.

**Comment by matty004** [comment]
Hey Hannah!
First thanks for this code, it was alot of fun to put together the mirror. 
Is there a way to revive the dead code for this? The MTA in New York released its real train time API and I was hoping to put the A,C,1 trains times on the mirror. Would I be better off starting from square 1? I know very little coding, but I'm learning so I was hoping some of the roots would be there.
